low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,9 +23,24 @@ import sys
 import socket
 import json
 import subprocess
-# Add the top level directories
-sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
-from framework.base.sspl_constants import PRODUCT_FAMILY, node_key_id, cluster_id, RESOURCE_HEALTH_VIEW
```

Comment:
cluster_id and node_key_id are not available from sspl_constants.py
So we get these these Conf.